test/TickMath.t.sol

```diff
@@ -44,6 +44,20 @@ contract TickMathTestTest is Test, JavascriptFfi {
         assertEq(maxTick, MAX_TICK);
     }
 
+    function test_maxUsableTick() public {
+        assertEq(tickMath.maxUsableTick(1), tickMath.MAX_TICK());
```

Comment:
these numbers feel a bit random. these are the fees in v3, but not the tick-spacings.

maybe we could add a few more?